backend/src/stats.js

```diff
@@ -253,26 +253,18 @@ aggregateActiveContractsCountByDate = retriable(
 );
 
 async function aggregateUniqueDeployedContractsCountByDate() {
-  const contractList = await queryUniqueDeployedContractsAggregatedByDate();
-  const contractsRows = contractList.map(
-    ({ date: dateString, deployed_contracts_by_date }) => ({
-      date: formatDate(new Date(dateString)),
-      deployedContracts: deployed_contracts_by_date,
-    })
-  );
-  let cumulativeContractsDeployedByDate = new Array();
-  let contractsDeployed = new Set([contractsRows[0].deployedContracts]);
-
-  for (let i = 1; i < contractsRows.length; i++) {
-    if (contractsRows[i].date !== contractsRows[i - 1].date) {
-      cumulativeContractsDeployedByDate.push({
-        date: contractsRows[i - 1].date,
-        contractsCount: contractsDeployed.size,
-      });
-    }
-    contractsDeployed.add(contractsRows[i].deployedContracts);
+  const contractsCountByDate = await queryUniqueDeployedContractsCountAggregatedByDate();
+  let cumulativeContractsCountByDate = [];
+  let cumulative_sum = 0;
+
+  for (let i = 0; i < contractsCountByDate.length; i++) {
+    cumulative_sum += contractsCountByDate[i].contracts_count_by_date;
+    cumulativeContractsCountByDate.push({
+      date: formatDate(new Date(contractsCountByDate[i].date)),
+      contractsCount: cumulative_sum,
+    });
```

Comment:
Actually, we usually build the cumulative data on the front-end side: https://github.com/near/near-explorer/blob/master/frontend/src/components/stats/NewContractsByDate.tsx#L25

That way we always return non-cumulative metrics from the backend, and be able to shape it into whatever thing we need on the frontend.

I won't block us on resolving it, but please, consider sending another PR fixing this situation.

Reply:
Oh, it's much easier
Thank you, will fix that